packages/checkout/sdk/src/types/smartCheckout.ts

```diff
@@ -410,14 +410,14 @@ export type NativeGas = {
 /**
  * Represents an ERC20 gas token.
  * @property {GasTokenType} type
- * @property {string} contractAddress
+ * @property {string} tokenAddress
  * @property {BigNumber} limit
  */
 export type ERC20Gas = {
   /** The type to indicate this is an ERC20 gas token. */
   type: GasTokenType.ERC20,
   /** The contract address of the ERC20. */
-  contractAddress: string;
+  tokenAddress: string;
```

Comment:
just reminder if we are gonna make this change to update the docs https://docs.immutable.com/docs/zkEVM/products/checkout/sdk/smart-checkout/smart-checkout-sell